usr/src/cmd/bhyve/pci_virtio_block.c

```diff
@@ -123,9 +154,14 @@ struct vtblk_config {
 struct virtio_blk_hdr {
 #define	VBH_OP_READ		0
 #define	VBH_OP_WRITE		1
+#define VBH_OP_SCSI_CMD		2
+#define VBH_OP_SCSI_CMD_OUT	3
 #define	VBH_OP_FLUSH		4
 #define	VBH_OP_FLUSH_OUT	5
 #define	VBH_OP_IDENT		8
+#define	VBH_OP_DISCARD		11
+#define VBH_OP_WRITE_ZEROES	13
```

Comment:
Fixed

Reply:
Once this exists upstream, we should match upstream as much as possible to minimize the effort of resyncs.  Including warts.

While it is still in review upstream, maybe it would be good to add this nit.  Then again, maybe it would just upset folks about us raising nits.

Reply:
@papertigers can correct me here, but I think the plan was since Alan's been so busy was we were going to get it into our Bhyve, then pick up the upstream effort (since I think both groups would like to see it get in).

Reply:
Yeah, the plan would be to drive the integration of this into FreeBSD as well.  Like Patrick did with the vlapic work etc.